packages/salesforcedx-vscode-core/src/commands/util/sfdxCommandlet.ts

```diff
@@ -165,6 +166,9 @@ export class SfdxCommandlet<T> {
   }
 
   public async run(): Promise<void> {
+    if (sfdxCoreSettings.getEnableClearOutputBeforeEachCommand()) {
```

Comment:
Adding this to the run() method here basically adds this feature to all the commands that extend `SfdxCommandlet` in the `salesforcedx-vscode-core` extension.  I wish there was an elegant single cut-point elsewhere in the `salesforcedx-utils-vscode` module like this, but there was not.